components/ai_chat/common/mojom/ai_chat.mojom

```diff
@@ -100,6 +100,8 @@ interface PageHandler {
   ClearConversationHistory();
   RetryAPIRequest();
   GetAPIResponseError() => (APIError error);
+  LikeMessage(bool is_liked, string turn_text) => (string? id);
+  SendFeedback(string category, string feedback, string rating_id);
 };
```

Comment:
As discussed, we can have callback here with request result so that we can show success / fail

Reply:
Triggering alert inside the callback now